crates/forge/tests/e2e/clean.rs

```diff
@@ -21,7 +21,8 @@ struct CleanComponentsState {
 }
 
 #[test]
-#[cfg_attr(not(feature = "scarb_2_8_3"), ignore)]
+#[cfg(not(target_os = "windows"))]
+#[ignore = "TODO(#2979)"]
```

Comment:
If these are broken, we should revert the PR introducing this logic before the release so we don't publish a broken feature.

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/2988